substrate/client/network/src/litep2p/mod.rs

```diff
@@ -1074,7 +1074,13 @@ impl<B: BlockT + 'static, H: ExHashT> NetworkBackend<B, H> for Litep2pNetworkBac
 							metrics.pending_connections_errors_total.with_label_values(&["transport-errors"]).inc();
 						}
 					}
-					_ => {}
+					None => {
+						log::error!(
```

Comment:
This comes only when it is terminated and not shutdown gracefully?

Reply:
Double checked with a stopped polkadot node and litep2p backend, we don't seem to emit errors here